base/applications/network/dwnl/dwnl.c

```diff
@@ -336,23 +344,24 @@ get_display_url(IN LPURL_COMPONENTS purl,
 }
 
 static int
-download_file(IN LPCTSTR pszUrl,
-              IN LPCTSTR pszFile  OPTIONAL)
+download_file(IN LPCWSTR pszUrl,
+             IN LPCWSTR pszFile OPTIONAL)
```

Comment:
One space character is missing for proper indentation.

Reply:
Fixed.